src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/on_system_request_notification.cc

```diff
@@ -138,7 +139,8 @@ void OnSystemRequestNotification::Run() {
 }
 
 #ifdef PROPRIETARY_MODE
-void OnSystemRequestNotification::AddHeader(BinaryMessage& message) const {
+void OnSystemRequestNotification::AddHeader(BinaryMessage& message,
```

Comment:
External proprietary won't get the full app id flag because it does not call add header.